packages/apps/spaces/ui/form/RichTextEditor/menu/BasicEditorToolbar.tsx

```diff
@@ -15,12 +15,7 @@ export const BasicEditorToolbar: FC<{
   const active = useActive();
 
   return (
-    <Flex
-      justifyContent='space-between'
-      alignItems='flex-end'
-      flex={1}
-      minH={8}
-    >
+    <Flex justifyContent='space-between' alignItems='center' flex={1} minH={8}>
```

Comment:
💅 Long lines of code are harder to read. I've tested on my local (checked if my eslint points to right config) and those lines are breaking after running fix command

Reply:
I don't think so. I'm formatting the code in this manner on save since I joined the project.